### ISSUE TITLE
Missing Hurstville area applications

### ISSUE THREAD

**Issue by equivalentideas** [LoveMyData/georges_river#1]
Hey @LoveMyData awesome to have one of these new councils running 👍 

So this new council is merged from Kogarah and Hurstville councils. This scraper is collecting the Kogarah ones, but doesn't seem to be getting the Hurstville ones http://daenquiry.hurstville.nsw.gov.au/masterviewui/Modules/Applicationmaster/default.aspx?page=found&1=thismonth&4a=DA%27,%27S96Mods%27,%27Mods%27,%27Reviews&6=F

I got sent to the old Hurstville site from the Georges River site http://www.georgesriver.nsw.gov.au/BUILDING/Planning-Proposals

While the [current Hurstville scraper](https://www.planningalerts.org.au/authorities/hurstville) continues to work , the [Kogarah one isn't](https://www.planningalerts.org.au/authorities/kogarah)—so this new scraper would get that going for people again. Should we just get this going and set the old Kogarah one to manual? Then when you have time get this scraper to pull data from the Hurstville one as well (you could just use the morph.io api for that)?

Does that make sense?

**Comment by LoveMyData** [comment]
@equivalentideas, I have already set old kogarah scraper to manual last week so it is not wasting resources, this is why I wrote this scraper

I don't like the idea using the API to pull data and put it inside Georges River council scraper. We are basically mashing morph database and generate more data.

Beside, we don't know what is the plan of both councils, they may start duplicating DA between their site too, so there will be more data from them and we mash our database via API, I think it will become really messy.

I hope this make sense.
